src/services/gapi.js

```diff
@@ -0,0 +1,36 @@
+/* global gapi */
+import $S from 'scriptjs';
+import once from 'lodash/once';
+import config from '../config';
+
+export const loadGapi = once(async() => new Promise((resolve) => {
+  $S('https://apis.google.com/js/client.js', async() => {
+    resolve(gapi);
+  });
+}));
+
+export const getGapi = once(async() => {
+  let gapi = await loadGapi();
+  gapi = await new Promise((resolve, reject) => {
```

Comment:
Is there a reason to reassign `gapi` here?

Reply:
nope - I think I miss understood how it was working originally